fili-core/src/main/java/com/yahoo/bard/webservice/table/PermissiveConcretePhysicalTable.java

```diff
@@ -44,7 +47,15 @@ public PermissiveConcretePhysicalTable(
                 timeGrain,
                 columns,
                 logicalToPhysicalColumnNames,
-                new PermissiveAvailability(name, columns, dataSourceMetadataService)
+                new PermissiveAvailability(name, dataSourceMetadataService)
         );
     }
+
+    @Override
+    public SimplifiedIntervalList getAvailableIntervals(DataSourceConstraint constraint) {
+        return getAvailability().getAvailableIntervals(PhysicalDataSourceConstraint.buildWithSchemaUnion(
+                constraint,
+                getSchema()
+        ));
```

Comment:
This should wrap differently:

```java
return getAvailability().getAvailableIntervals(
        PhysicalDataSourceConstraint.buildWithSchemaUnion(constraint, getSchema())
);
```

In general, the outer-most param list is where the wrapping should occur, rather than the inner-most, in a case like this where there are nested calls.